bin/build-all-plugins.sh

```diff
@@ -1,19 +1,14 @@
 #!/bin/sh
 
```

Comment:
The big change with this commit is that, instead of having a single `plugins/index.js` file that requires `leveldown` and gets swapped out for the appropriate *down adapter using command-line Browserify arguments, instead I've just got three different `index.js` files, one for each of memory/idb-alt/localstorage.